source/Misc.hpp

```diff
@@ -826,7 +829,7 @@ typedef struct _FailProps
 const int nCoef = 30; // maximum number of entries to allow in nonlinear
                       // coefficient lookup tables
 
-typedef struct
+struct EnvCond
```

Comment:
In general the typedef is preferred, right? Basically because in C this way you have to refer to this type as `struct EnvCond` instead of `EnvCond` directly

Reply:
The `struct EnvCond` syntax for declaring a struct in c++ behaves nearly exactly the same as `class EvnCond` and you don't have to prefix the type with struct to use it. The other advantage of this syntax is that it allows forward declarations which happens in WaveGrid.hpp to avoid some unnecessary imports there which may help compilation in some cases.

Reply:
Your point seems perfectly reasonable